tests/cnf/core/network/metallb/tests/bgp-gracefulrestart.go

```diff
@@ -205,6 +373,59 @@ func gracefulRestartTest(
 	Expect(err).ToNot(HaveOccurred(), "Unexpectedly succeeded in curling the Nginx pod")
 }
 
+func gracefulRestartTestWithBFD(
+	extTrafficPolicy corev1.ServiceExternalTrafficPolicyType,
+	destIPaddress,
+	ipv4metalLbIP string,
+	frrPod *pod.Builder,
+	ipAddressPool *metallb.IPAddressPoolBuilder,
+	workerLabelMap map[string]string) {
+	By("Creating a MetalLB service")
+	setupMetalLbService(netparam.IPV4Family, ipAddressPool, extTrafficPolicy)
+
+	By("Checking that routes have been added to the BGP table")
+	verifyRoutesAmount(frrPod, 1)
```

Comment:
Is there a reason why not the exact route?

Reply:
Just for simplicity.
Any reason for checking exact route?

Reply:
Is safer it can give you less false positive, it might enable parallel testing, not sure is more complicated.